frontend/src/components/auth/SignUpForm.vue

```diff
@@ -151,9 +153,11 @@
                         email: this.newUser.email,
                         password: this.newUser.password,
                     }).then(() => {
+                        this.$emit("success");
+                        this.showSuccessMessage('You have been successfully registered! Please log in!');
                         this.$router.push({name: 'login'});
-                    }, err => {
-                        this.showErrorMessage(err.message);
+                    }).catch((error) => {
+                        this.showErrorMessage(_.get(error, 'response.data.error.message', 'Unknown error'));
```

Comment:
I'm going to approve it, but this needs to be changed, since it's not the responsibility of component, error should be formatted by action

Reply:
Done!